docs-site/package.json

```diff
@@ -38,7 +39,7 @@
     ]
   },
   "engines": {
-    "node": ">=16.14"
+    "node": ">=20.14.0"
   },
   "pnpm": {
```

Comment:
**nitpick (blocking):** Can you try removing this section, not sure but you may need to update to the latest version of pnpm in `.tool-versions`.

Reply:
Updated in 276320f4653f589781ce3dfc58ec9af8a1aec1e0